98_exercises.Rmd

```diff
@@ -54,7 +64,7 @@ Functions that might be useful: wilcox.test, kruskal.test, ggplot, pheatmap, Com
 
 Functions that might be useful: runMDS, runRDA, anova.cca, transformSamples, agglomerateByRank, ggplot, plotReducedDim
 
-## Introduction to MAE
+## Introduction to MAE {.unlisted .unnumbered}
```

Comment:
Shall we use full names + acronym like MultiAssayExperiment (MAE). Could be more clear?